### ISSUE TITLE
Original docs reference

### ISSUE THREAD

**Issue by JajarGaming** [DottieDot/GTAV-NativeDB#69]
Would be great if each native could include the place it was grabbed from as I've wanted to contribute to some of them like `FORCE_PED_MOTION_STATE` which has no decents docs, but idk which website it was taken from

**Comment by DottieDot** [comment]
I agree. But for now, besides the FiveM natives and the script examples, all native data is coming from https://github.com/alloc8or/gta5-nativedb-data.
